cmd/manager/main.go

```diff
@@ -283,7 +283,9 @@ func printConfig(cfg *configuration.Config) {
 
 	logWithValuesHost := log.WithValues(
 		"key", getHostEnvVarKey(configuration.VarDurationBeforeChangeRequestDeletion),
-		"value", cfg.GetDurationBeforeChangeTierRequestDeletion())
+		"value", cfg.GetDurationBeforeChangeTierRequestDeletion(),
+		"key", getHostEnvVarKey(configuration.VarRegistrationServiceURL),
+		"value", cfg.GetRegistrationServiceURL())
```

Comment:
Two "key"s and two "value"s? It doesn't look right.

Reply:
@alexeykazakov is this better: https://github.com/codeready-toolchain/host-operator/pull/257/commits/523839e0a56a02047c39b1ae8157524de6c7cc9b#diff-9e63f7b0791edcb4a53c944552b88970L287-L288